docs/index.rst

```diff
@@ -262,158 +260,22 @@ Divider looks
 209. Shrimp
 
 
-
 And more coming soon!  
 
-Installing  
-==========
-
-*You may install in one of the following ways:*  
-
-- Through pip  
-- Through requirements.txt  
-
-To install via pip  
-------------------
-To install via pip, open a terminal, and type the following command:  
-
-.. code-block:: console
-
-    $ pip install area4 
-
-It should install.  
-
-To install via requirements.txt  
--------------------------------
-
-To use area4 as a dependency for your project, you can add the following line:  
-
-.. code-block:: console
-
-    area4 
-
-You must have prior knowledge with using a requirements.txt file to take this path.  
```

Comment:
Maybe we should add more info on using the requirements file, not all people know a lot about pip.